defs/scenarios/storage/ceph/ceph-osd/PG_overdose.yaml

```diff
@@ -0,0 +1,26 @@
+checks:
+  health_warning:
+    property:
+      path: hotsos.core.plugins.storage.ceph.CephCluster.health_status
+      ops: [[ne, HEALTH_OK]]
+  # this one will run on OSD sos
+  ceph_osd_withhold_creation:
+    input:
+      path: var/log/ceph/ceph*.log
```

Comment:
be careful using such a permissive path since this can lead to search some files unnecessarily. if you want to look at only ceph-mon.log or ceph.log you can do e.g.

input: [var/log/ceph/ceph.log, var/log/ceph/ceph-mon.log]

Reply:
ack, done. 

```
  ceph_osd_withhold_creation:
    input:
      path: [var/log/ceph/ceph*osd*.log, var/log/ceph/ceph.log]
    search:
      expr: '.+ (osd.+) .+ maybe_wait_for_max_pg withhold creation of pg .+: (.+) >= (.+)'
```